assets/src/components/map.tsx

```diff
@@ -158,6 +160,74 @@ export const updateMarkers = (
   )
 }
 
+const makeTrainVehicleIcon = ({ bearing }: TrainVehicle): Leaflet.DivIcon => {
+  const centerX = 12
+  const centerY = 12
+  return Leaflet.divIcon({
+    html: `<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 36 36">
+        <g transform="rotate(${bearing}, ${centerX}, ${centerY})">
+          <path fill="#fff" d="m42.88 45.83a2.1 2.1 0 0 1 -.87-.19l-15.92-7.17a5.23 5.23 0 0 0 -2.09-.47 5.14 5.14 0 0 0 -2.08.44l-15.92 7.2a2.1 2.1 0 0 1 -.87.19 2.14 2.14 0 0 1 -1.76-1 2 2 0 0 1 -.12-2l18.86-40.83a2.08 2.08 0 0 1 3.78 0l18.87 40.87a2 2 0 0 1 -.12 2 2.14 2.14 0 0 1 -1.76.96z"/>
+        </g>
+    </svg>`,
+    iconAnchor: [centerX, centerY],
+    className: "m-vehicle-map__train-icon",
+  })
+}
+
+export const updateTrainVehicleMarkers = (
+  newTrainVehicles: { [id: string]: TrainVehicle },
```

Comment:
```suggestion
  newTrainVehicles: TrainVehiclesById,
```

since that type alias is used elsewhere.